packages/RichTextEditing-Core.package/TextStructure.class/instance/amount.st

```diff
@@ -1,4 +1,6 @@
 accessing
 amount
-"A textIndent attribute has to provide a amount method specifying the indentationAmount"
-	^ 2
\ No newline at end of file
+
```

Comment:
I think a comment is appropriate here, as you can not easily tell why the word amount is used